.storybook/preview.js

```diff
@@ -1,3 +1,7 @@
+import React from "react"
+import { addDecorator } from "@storybook/react"
+import { useEffect } from "@storybook/addons"
```

Comment:
Well, this is a thing I didn't know about

Reply:
Good point, I can't even find the docs I copy-pasted this from. 

Why not use React's `useEffect`? It seems to work fine! I have updated this accordingly